vipps-order-management-api.md

```diff
@@ -22,144 +16,48 @@ API version: 1.0.3.
-  status of the order. If your webshop or site has a "My orders" page or
-  similar - this link category can take the customer there.
-* **Delivery information**: A link to a location that contains information and
-  status about the shipping or delivery related to the order. This could be a
-  link to hosted by your freight carrier, or a link to your site. If your
-  webshop or site has a "My order" page that includes delivery related
-  information about the order - this link category can be used.
-* **Ticket**: A link to a location where the customer can access and download a
-  ticket to an event, trip or transportation.
-* **Booking**: A link to a location that contains information and status about a
-  booking, such as travel and rental booking. If your webshop or site has a
-  "My bookings" page or similar, this link category can take the customer there.
-* **General**: If none of the other categories fit the use case for the link,
-  a *General* category can be used. This is a link to a location that contains
-  any kind of information pertinent to the order. We encourage you to use the
-  more specific categories if possible.
-
-### Image
-
-With Vipps Order Management API you can upload an image that is shown on the
-transaction in the Vipps app. To add an image to an order you first need to use
-the [`POST:/images`](https://vippsas.github.io/vipps-order-management-api/#/images/postImage) endpoint to upload an image. Thereafter, the newly uploaded
-image's `imageId` can be added using the `orderInfo` API. The same image may
-be used for multiple transactions, but uploading a unique image for each
-transactions is also OK. Imaged are fetched authenticated from the app, so feel
-free add tickets and receipts as images. Below you can see an example of a Vipps
-receipt containing an image of the shopping card (single product)
-<p align="center">
-  <img src="images/order-link-shipping-information-with-image.png"
-       alt="Shipping information link" width="150">
-</p>
-
-## Receipt
-
-In addition to providing a user with a link to a valid receipt hosted on your
-site, it is also possible have the receipt hosted inside the Vipps app. To
-enable this you can send necessary information such as order lines and VAT rates
-to the Order Management API and Vipps will generate a receipt that can be used
-for proof of purchase and expensing. This is done using the [`POST:/receipts`](https://vippsas.github.io/vipps-order-management-api/#/receipts/postReceipt) endpoint.
-
-In addition to providing the user with a valid receipt inside Vipps, order lines
-will also give the user a much better overview of the purchase. Furthermore, in
-the case of partial order fulfilments or returns the order lines may be updated
-to reflect what the user actually ended up paying for using Vipps.
-
-<p align="center">
-  <img src="images/orderlines.png" width="150">
-</p>
-
-## Getting Started
 
-### Before you begin
+# Before you begin
```

Comment:
This (and subsequent) will result in markdown error: [Multiple top-level headings in the same document](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md025) .

Reply:
I've been trying to fix all those :-)

Reply:
😮 
looks like its working though, but i should probably change it. Aa, i like the design of h1 headers 😢

Reply:
Oh, no problem! If you like it better, let's just keep it!
I've been changing them in several places, so I don't think I'll do that any more.